src/main/java/org/skife/jdbi/v2/Query.java

```diff
@@ -420,4 +426,13 @@ public void registerMapper(ResultSetMapperFactory m)
     {
         this.mappingRegistry.add(m);
     }
+
+    public void registerColumnMapper(ResultColumnMapper m)
```

Comment:
I think this needs to exist so that the upcoming `@RegisterColumnMapper` and `@RegisterColumnMapperFactory` annotations on query methods can be correctly scoped.